test/common/http/conn_pool_grid_test.cc

```diff
@@ -135,6 +137,8 @@ class ConnectivityGridTestBase : public Event::TestUsingSimulatedTime, public te
   std::shared_ptr<Upstream::MockClusterInfo> cluster_{new NiceMock<Upstream::MockClusterInfo>()};
   NiceMock<Random::MockRandomGenerator> random_;
   AlternateProtocolsCacheSharedPtr alternate_protocols_;
+  Stats::IsolatedStoreImpl scope_;
```

Comment:
I think typically in tests if we are instantiating a store we'd call the member variable `stats_` or `store_` or `stats_store_`.

Here & below.